src/lib.rs

```diff
@@ -97,11 +99,11 @@ impl Generator {
     /// ```
     pub fn new_composite(n: usize) -> BigUint {
         let mut rng = rand::thread_rng();
+        let n = n.try_into().unwrap();
+
         loop {
-            // Make mutable and set LSB and MSB
-            let candidate: BigUint = rng.gen_biguint(n);
-            //candidate.set_bit(0, true);
-            //candidate.set_bit((n-1) as u32, true);
+            let mut candidate: BigUint = rng.gen_biguint(n);
+            candidate.set_bit((n-1) as u64, true);
```

Comment:
Composite numbers should be able to be even(?) so I didn't set the 0th bit. Let me know if we only want to generate odd composites.